### PR TITLE
Outside us v2

### PR DESCRIPTION
**Pull request by toshokanneko**
Disable ads outside the US (again).

### CODE REVIEW

File: web/component/ads/view.jsx [hunk 1]
```diff
@@ -144,7 +145,7 @@ function Ads(props: Props) {
   );
 
   if (type === 'video') {
-    if (shouldShowAds) {
+    if (shouldShowAds && userIsInUS) {
```
Comment (infinite-persistence) [inline web/component/ads/view.jsx:148]: Perhaps the mistake is in `resolveAdVisibility`, and should be corrected there? `shouldShowAds` should cover all scnearios.

Comment (toshokanneko) [reply web/component/ads/view.jsx:148]: I don't know. Your solution seems to work here. I'm not quite sure what's happening. Anthony mentioned that he sees the ads again. This PR seems unnecessary and I might delete it. I'm just trying to understand why it isn't working for him...

Comment (toshokanneko) [reply web/component/ads/view.jsx:148]: My bad. I misread that line.

Comment (infinite-persistence) [reply web/component/ads/view.jsx]: Throwing out ideas:
`export const selectUserCountry = createSelector(selectUserLocale, (locale) => locale?.country);`
means `country` could be `undefined` until fetched, so `undefined !== 'US'` results in `true`?

Comment (infinite-persistence) [reply web/component/ads/view.jsx]: Ah, never mind, saw the update.

Comment (toshokanneko) [reply web/component/ads/view.jsx]: Should be fixed now :D Gonna test all the potential scenarios...

**Pull request merged**